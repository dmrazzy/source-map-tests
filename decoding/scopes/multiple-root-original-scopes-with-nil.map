{
  "version": 3,
  "file": "multiple-root-original-scopes-with-nil.js",
  "sources": [
    "original_0.js",
    "original_1.js",
    "original_2.js"
  ],
  "mappings": "",
  "names": [
    "global"
  ],
  "scopes": "BCAAA,CKA,A,BCAAA,CKA"
}