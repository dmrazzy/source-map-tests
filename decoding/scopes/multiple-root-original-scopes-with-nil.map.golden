{
  "file": "multiple-root-original-scopes-with-nil.js",
  "mappings": [],
  "sources": [
    {
      "url": "original_0.js",
      "content": null,
      "ignored": false,
      "scope": {
        "start": {
          "line": 0,
          "column": 0
        },
        "end": {
          "line": 10,
          "column": 0
        },
        "name": null,
        "kind": "global",
        "isStackFrame": false,
        "variables": [],
        "children": []
      }
    },
    {
      "url": "original_1.js",
      "content": null,
      "ignored": false,
      "scope": null
    },
    {
      "url": "original_2.js",
      "content": null,
      "ignored": false,
      "scope": {
        "start": {
          "line": 0,
          "column": 0
        },
        "end": {
          "line": 10,
          "column": 0
        },
        "name": null,
        "kind": "global",
        "isStackFrame": false,
        "variables": [],
        "children": []
      }
    }
  ],
  "ranges": []
}